specification/nginx/resource-manager/NGINX.NGINXPLUS/preview/2025-03-01-preview/swagger.json

```diff
@@ -0,0 +1,2850 @@
+        "attackSignaturesPackage": {
+          "$ref": "#/definitions/WebApplicationFirewallPackage",
+          "readOnly": true,
+          "description": "Package containing attack signatures for the NGINX App Protect Web Application Firewall (WAF)."
+        },
+        "botSignaturesPackage": {
+          "$ref": "#/definitions/WebApplicationFirewallPackage",
+          "readOnly": true,
+          "description": "Package containing bot signatures for the NGINX App Protect Web Application Firewall (WAF)."
+        },
+        "threatCampaignsPackage": {
+          "$ref": "#/definitions/WebApplicationFirewallPackage",
+          "readOnly": true,
+          "description": "Package containing threat campaigns for the NGINX App Protect Web Application Firewall (WAF)."
+        },
+        "componentVersions": {
+          "$ref": "#/definitions/WebApplicationFirewallComponentVersions",
+          "readOnly": true,
+          "description": "Versions of the NGINX App Protect Web Application Firewall (WAF) components."
+        }
+      }
+    },
+    "WebApplicationFirewallPackage": {
+      "type": "object",
+      "description": "NGINX App Protect Web Application Firewall (WAF) Package. Contains the version and revision date of the package.",
+      "required": [
+        "version",
+        "revisionDatetime"
+      ],
+      "properties": {
+        "version": {
+          "type": "string",
+          "example": "2.02",
+          "description": "The version of the NGINX App Protect Web Application Firewall (WAF) package."
+        },
+        "revisionDatetime": {
+          "type": "string",
+          "example": "2024-02-21T15:50:53Z",
+          "format": "date-time",
+          "description": "The date and time of the package revision."
+        }
+      }
+    },
+    "WebApplicationFirewallComponentVersions": {
+      "type": "object",
+      "readOnly": true,
+      "required": [
+        "wafEngineVersion",
+        "wafNginxVersion"
+      ],
+      "description": "Versions of the NGINX App Protect Web Application Firewall (WAF) components.",
+      "properties": {
+        "wafEngineVersion": {
+          "type": "string",
+          "example": "10.624.0",
+          "description": "The version of the NGINX App Protect Web Application Firewall (WAF) engine."
+        },
+        "wafNginxVersion": {
+          "type": "string",
+          "example": "4.815.0",
+          "description": "The version of the NGINX App Protect Web Application Firewall (WAF) module for NGINX."
+        }
+      }
+    },
+    "NginxDeploymentWafPolicy": {
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "title": "Nginx Deployment Waf Policy",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true
+        },
```

Comment:
these are implied if you inherit proxyresource

Reply:
Same as above